### ISSUE TITLE
Auth0 actually maintaining this?

### ISSUE THREAD

**Issue by dbinetti** [auth0/auth0-ember-simple-auth#53]
Can someone from Auth0 indicate if you are actually maintaining this library?  No activity since August...

**Comment by dbinetti** [comment]
I put a request to support. NR yet

**Comment by seawatts** [comment]
You should take a look at https://github.com/seawatts/ember-simple-auth-auth0

**Comment by chenkie** [comment]
Hey sorry for the long delay here, we're in the process of migrating items over to a new org called auth0-community, and Ember falls into this camp. It looks like @seawatts has a great integration which we'll take a look at and see if it can be introduced into our samples and quickstart.

**Comment by dbinetti** [comment]
FWIW, I spoke with Eugenio about this at length and he was supportive of
merging in @seawatts code -- and @seawatts also expressed willingness to
help with the integration in this regard.

**Comment by seawatts** [comment]
@chenkie @dbinetti yes I am definitely willing to help migrate this over =) Let me know how I can help. =)

**Comment by seawatts** [comment]
Linking to #57

**Comment by mwisner** [comment]
Now that https://github.com/auth0-community/ember-simple-auth-auth0 exists i'm wondering what should be used? 

Should this project be deprecated in favor of https://github.com/auth0-community/ember-simple-auth-auth0?

**Comment by dbinetti** [comment]
those are the same repos

**Comment by mwisner** [comment]
Sorry for the confusion, I did not see the deprecation notice in the readme of this repo.
